server/src/validators/space.validator.js

```diff
@@ -0,0 +1,25 @@
+const { check, validationResult } = require("express-validator");
+
+const validateSpaceRequest = [
+    check("spaceName")
+    .notEmpty()
+    .withMessage("spaceName is required"),
+    check("description")
+    .notEmpty()
+    .withMessage("description is Required")
+
```

Comment:
also add condition for `createdBy` not empty

Reply:
createdBy should be bi-default until we have the admin access, We don't need to add createdBy, at this time. I have mentioned in the schema default value is the user. so he we have the admin panel then it would be necessary